### PR TITLE
Added wrapper function and tests for wrapper function

### PR DESCRIPTION
**Pull request by sweber15**
Added wrapper function and tests for wrapper function

### CODE REVIEW

**Review by sgauravm — APPROVED** [review]
Looks good. You can merge it

File: tests/testthat/test-eda.R [hunk 1]
```diff
@@ -1,10 +1,38 @@
...
+    expect_true(bad_results == FALSE)
+  })
+  
+  # Tests that TRUE is returned when there is no error
+  test_that("Tests that TRUE is returned when there is no error", {
+    expect_true(results == TRUE)
+  })
+}
+
+test_generate_report()
```
Comment (sgauravm) [inline tests/testthat/test-eda.R:27]: you can remove this line

**Review by sgauravm — CHANGES_REQUESTED** [review]

File: tests/testthat/test-eda.R (hunk 1 above)
Comment (marvinmin) [reply tests/testthat/test-eda.R:27]: No, it is actually needed to run the test.

Comment (sweber15) [reply tests/testthat/test-eda.R:27]: I thought we should be calling the test in the test file.

**Review by marvinmin — APPROVED** [review]
Looks good.

File: tests/testthat/test-eda.R (hunk 1 above)
Comment (sgauravm) [reply tests/testthat/test-eda.R:27]: Yes call the function

**Review by sgauravm — COMMENTED** [review]
After adding the line you removed . It can  be merged

**Review by sgauravm — APPROVED** [review]

**Pull request merged**